Tests/Approval.Tests/ApprovedApi/FluentAssertions/net47.verified.txt

```diff
@@ -67,7 +69,6 @@ namespace FluentAssertions
         public static FluentAssertions.Specialized.NonGenericAsyncFunctionAssertions Should([System.Diagnostics.CodeAnalysis.NotNull] this System.Func<System.Threading.Tasks.Task> action) { }
         public static FluentAssertions.Primitives.GuidAssertions Should(this System.Guid actualValue) { }
         public static FluentAssertions.Primitives.NullableGuidAssertions Should(this System.Guid? actualValue) { }
-        public static FluentAssertions.Streams.BufferedStreamAssertions Should([System.Diagnostics.CodeAnalysis.NotNull] this System.IO.BufferedStream actualValue) { }
```

Comment:
I'm wondering whether is change is related to the AssertionScope feature.
This also applies to the removal of `BufferedStreamAssertions`.
I did not find this in your PR description.